docker/sync_repo

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+  
+REPO=$1
```

Comment:
We need an option for a branch too, right?

Reply:
Yep, just keeping it simple to test the fix